lib/service/locale_service.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:io';
+import 'package:autonomy_flutter/common/injector.dart';
+import 'package:autonomy_flutter/service/navigation_service.dart';
+import 'package:autonomy_flutter/util/geolocation.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/services.dart';
+
+class LocaleService {
+  static const MethodChannel _channel = MethodChannel('locale_measurement');
+  static GeoLocation? geolocation;
+  static String measurementSystem = "metric";
+
+  static Future<void> getLocaleMeasurement() async {
```

Comment:
suggestion to rename, this function will refresh the locale data. Simply can name it `refresh()`